jin/4주차 과제/9-기초비트단위논리연산.py

```diff
@@ -0,0 +1,19 @@
+# 입력 된 정수를 비트단위로 참/거짓을 바꾼 후 정수로 출력
+bitNot = ~int(input())
+print( bitNot )
```

Comment:
```suggestion
print( ~int(input())
```
이렇게 간략하게 짤 수도 있을 것 같아요!